transport/http/inbound_example_test.go

```diff
@@ -83,3 +84,52 @@ func ExampleMux() {
 	}
 	// Output: hello from /health
 }
+
+func ExampleInterceptor() {
+	// import nethttp "net/http"
+
+	// We create an interceptor which executes an override http.Handler if
+	// the HTTP request is not a well-formed YARPC request.
+	override := func(overrideHandler nethttp.Handler) http.InboundOption {
+		return http.Interceptor(func(transportHandler nethttp.Handler) nethttp.Handler {
+			return nethttp.HandlerFunc(func(w nethttp.ResponseWriter, r *nethttp.Request) {
+				if r.Header.Get("RPC-Encoding") == "" {
+					overrideHandler.ServeHTTP(w, r)
+				} else {
```

Comment:
For the example we're showing to users, I would much rather have it be extremely simple. Something that simply logs a specific header or similar. Let's not demo overrides as an example.

Reply:
I hear you, but I think I'd like to roll with this example: it's slightly complicated, but at least illustrates a real usecase, and I like the fact it's the actual usecase that we have.